docs/advanced-topics/organizations-and-collections.rst

```diff
@@ -90,18 +90,18 @@ can request a re-review.
 Organization membership
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Anyone can see an approved organization, though only members and
-maintainers of the organization can perform actions on the
-organizations. This includes creating and adding to collections,
-updating metadata, and adding new members.
+Anyone can see an approved organization, but only admins and
+maintainers of the organization can perform actions on it.
+This includes creating and adding to collections, updating metadata, and adding new members.
 
-There are two types of roles available:
+As of 1.9.2, there are three types of roles available:
```

Comment:
Doesn't our documentation versioning take care of this so we don't have to mention the version?

Reply:
It can take care of versioning, but we don't yet do it that way. But maybe we should do it for releases.

Reply:
I think there is a story or issue to start tagging our doc releases, isn't there?

Reply:
Hm...I couldn't find one. I know we've discussed it before.